src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -122,15 +105,59 @@
 	}
 }
 
-/* Theme specific overrides */
+/* = Organizer block view styles
+============================================= */
+.tribe-block__organizer__details {
+	border-top: 1px solid #d5d5d5;
```

Comment:
are we using variables or hardcoded colors?

Reply:
I've added a variable here

Reply:
there are more colors that need to be changed to variables in this PR

Reply:
I've updated all the colors now